apps/privacy-panel/test/marionette/tc_main_test.js

```diff
@@ -86,6 +86,18 @@ marionette('transparency control panels', function() {
     assert.ok(find(app.chubbyU + ' ~ ' + app.flyingP));
     assert.ok(find(app.flyingP + ' ~ ' + app.mightyD));
```

Comment:
can you add 
```
    prefs: {
      'focusmanager.testmode': true
    },
```

to the options when initializing the `marionette.client`? otherwise the keyboard-related test will fail locally for some people